os/booting-on-azure.md

```diff
@@ -34,6 +34,30 @@ The following command will create a single instance. For more details, check out
   </div>
 </div>
 
+## Ignition Config
+
+CoreOS allows you to configure machine parameters, configure networking, launch systemd units on startup, and more via Ignition. Head over to the [docs to learn about the supported features][ignition-docs]. Note that Microsoft Azure doesn't allow an instance's userdata to be modified. This isn't a problem since Ignition only runs on the first boot.
```

Comment:
I think we need to add this, otherwise it reads as "you can't use ignition on azure"

> modified **after creation**.